cis194/week1/robert-chang/Homework01/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2019
```

Comment:
Should we have a repo-wide license instead?

Reply:
ohh. those are generated by stack. I will remove it then.

Reply:
done removed. please check again.